app/search.py

```diff
@@ -494,6 +497,78 @@ def _query_fields(self, fields):  # pylint: disable=too-many-locals
                 f'{fields} "{preNonDigitAndPreDigit}"~5',
                 f'{fields} "{infNonDigitAndPreDigit}"'
             ]
+
+        # to improve the fuzzyness of the search result the search text has to be changed
+        # digit/text combinations are replaced with (digit|digit/text) p.e. 4a -> (4|4a)
+        # quorum operatar is used for query text fuzzy matching
+        if fuzzySearch:
```

Comment:
Just wondering, if it could improve the readability of the code to move the methods some levels up.
Might be confusing to have methods defined inside an if-branch inside another method.

Reply:
done